lib/routes/netease/dy2.js

```diff
@@ -31,31 +31,16 @@ module.exports = async (ctx) => {
 
     const items = await Promise.all(
         list.map(async (item) => {
-            let content;
-            if (item.link.startsWith('https://www.163.com')) {
-                const itemData = await ctx.cache.tryGet(item.link, async () =>
-                    iconv.decode(
-                        (
-                            await got.get(item.link, {
-                                responseType: 'buffer',
-                            })
-                        ).data,
-                        charset
-                    )
-                );
-                content = cheerio.load(itemData, { decodeEntities: false });
-            } else {
-                const itemData = await ctx.cache.tryGet(
-                    item.link,
-                    async () =>
-                        (
-                            await got.get(item.link, {
-                                responseType: 'buffer',
-                            })
-                        ).data
-                );
-                content = cheerio.load(itemData, { decodeEntities: false });
-            }
+            const itemData = await ctx.cache.tryGet(
+                item.link,
+                async () =>
+                    (
+                        await got.get(item.link, {
+                            responseType: 'buffer',
+                        })
+                    ).data
```

Comment:
有运行过 lint 任务吗？

```suggestion
                () => (
                    got.get(item.link, {
                        responseType: 'buffer',
                    })
                ).data
```

https://github.com/DIYgod/RSSHub/pull/8066/files#diff-b51b5a6ac4c00d72e82e8d8577aea2a49610113480620c8546f385c39501deb8R45-R53

```
return {
    title: item.title,
    link: item.link,
    description: content('.post_body').html(),
    pubDate: item.pubDate,
};
```